packages/teleterm/README.md

```diff
@@ -137,6 +137,16 @@ See [Teleport Connect build
 process](https://gravitational.slab.com/posts/teleport-connect-build-process-fu6da5ld) on Slab for
 bulid process documentation that is specific to Gravitational.
 
+### Native dependencies
+
+If node-pty doesn't provide precompiled binaries for your system and the specific Electron version,
+you will need to install [the dependencies required by
+node-pty](https://github.com/microsoft/node-pty#dependencies).
+
+### Linux
+
+To create RPM package for aarch64 target you need to provide `USE_SYSTEM_FPM=1` env var.
```

Comment:
```suggestion
To create an RPM package for an aarch64 target you need to provide `USE_SYSTEM_FPM=1` env var.
```
I've never seen it called aarch64, TIL. Always thought people just used `arm64`.

Reply:
We tend to use arm64 within our codebases too so I'm going to rename it.